edb/testbase/lang.py

```diff
@@ -323,8 +323,8 @@ def run_ddl(cls, schema, ddl, default_module=defines.DEFAULT_MODULE_ALIAS):
                 ddl_plan = s_ddl.compile_migration(
                     ddl_plan, target_schema, current_schema)
 
-            elif isinstance(stmt, qlast.Delta):
-                # APPLY MIGRATION
+            elif isinstance(stmt, qlast.CommitMigration):
+                # COMMIT MIGRATION
```

Comment:
Please tell me one last thing about a detail for the connection pool. It's something I wanted to discuss and then forgot to ask ->

Reply:
Well, all the current tests that test migrations have been converted, there are plenty of them.